src/components/FoodBar/FoodBar.js

```diff
@@ -0,0 +1,38 @@
+import "./FoodBar.css";
+import { useContext, useEffect, useState } from "react";
+import { SessionContext } from "../../pages/App/App";
+
+export default function FoodBar() {
+	const [food, setFood] = useState("");
+
+	const { farmFood, setWinCondition } = useContext(SessionContext);
+
+	useEffect(() => {
+		// If winCondition(true) modal from RewardModal replaces the GameContainer elements on the screen
+		if (farmFood === 100) {
+			setWinCondition(true);
+		}
+		// Created a conditional to check if the food bar is empty, was getting a weird line rendered because of the CSS border setting. Removing the class if it is empty, and applying the class only when the bars height > 0 solves this.
```

Comment:
Long comments should be broken across multiple lines so you don't have to scroll horizontally to read the entire comment